editor.planx.uk/.eslintrc

```diff
@@ -73,5 +72,11 @@
   },
   "settings": {
     "jest": { "version": 27 }
-  }
+  },
+  "overrides": [
```

Comment:
This config update means that these linting rules will only apply to test files - I was getting a linting error from `testing-library/react` for using the `client.user.getById()` which shares a name with a `testing-library` function.